exercises/traits/traits5.rs

```diff
@@ -30,7 +28,7 @@ impl SomeTrait for OtherStruct {}
 impl OtherTrait for OtherStruct {}
 
 // YOU MAY ONLY CHANGE THE NEXT LINE
-fn some_func(item: ??) -> bool {
+fn some_func(item: (impl SomeTrait + OtherTrait)) -> bool {
```

Comment:
Why the parentheses?

Reply:
Just used to doing that when there is an operator between objects
removed it